get_gambl_results.smk

```diff
@@ -0,0 +1,80 @@
+# It also requires an RSA key that is passphrase-protected. Other key types don't seem to work.
+# You can create one and add it to your authorized_keys file if your current key is not passphase-protected
+
+
+hostname = 'gphost08.bcgsc.ca'
+myuser = 'rmorin' #UPDATE with your GSC username
+mypassphrase = 'GAMBL' #UPDATE with the passphrase for your private key. This only seems to work if you use a key that has a passphrase!
+mySSHK = '/Users/rmorin/.ssh/id_rsa2' # UPDATE set this to the path of your RSA private key if it has a different name (it probably will)
+
+from snakemake.remote.SFTP import RemoteProvider
+SFTP = RemoteProvider(username=myuser,private_key_pass=mypassphrase,private_key=mySSHK)
+
+configfile: "gambl/config.yml" #UPDATE: specify the full path to the GAMBLR config.yml or make a symlink so this points to it
+
+
+project_base = config["default"]["project_base"]
+repo_base = config["default"]["repo_base"]
+
+flatfiles = config["default"]["results_flatfiles"]
+
+resources = config["default"]["resources"]
+
+#here we specify which files are included from the GAMBLR config
+db_maf = flatfiles["ssm"]["template"]["cds"]["deblacklisted"]
+aug_maf = flatfiles["ssm"]["template"]["cds"]["augmented"]
+cnv_combined = flatfiles["cnv_combined"]["template"]
+sv_combined = flatfiles["sv_combined"]["template"]
+blacklist = resources["blacklist"]["template"]
+
+
+seq_types = list(config["default"]["unmatched_normal_ids"]["gambl"].keys())
+projections = config["default"]["projections"].split(",")
+
+
+
+rule all:
+    input: 
+        deblacklisted = expand(db_maf,seq_type=seq_types,projection=projections),
+        augmented = expand(aug_maf,seq_type=seq_types,projection=projections),
+        combined_cnv = expand(cnv_combined,seq_type=['genome'],projection=projections),
+        combined_sv = expand(sv_combined,seq_type=['genome'],projection=projections),
+        blacklists = expand(blacklist,seq_type=seq_types,projection=projections)
+
+#Use the relative directory for local file names (outputs) and full path for remote file names (inputs)
+rule get_mafs:
+    input:
+        db = SFTP.remote(hostname + project_base + db_maf),
+        aug = SFTP.remote(hostname + project_base + aug_maf),
+    output:
+        deblacklisted = db_maf,
+        augmented = aug_maf,
+    run:
+        shell("cp {input.db} {output.deblacklisted}")
+        shell("cp {input.aug} {output.augmented}")
+        
+rule get_resources:
+    input:
+        SFTP.remote(hostname + project_base + blacklist)
+    output:
+        blacklist
+    run:
+        shell("cp {input[0]} {output[0]}")
+
+rule get_cnv:
+    input:
+        cnv = SFTP.remote(hostname + project_base + cnv_combined)
+    output:
+        combined_cnv = cnv_combined
+    run:
+        shell("cp {input[0]} {output[0]}")
+
+rule get_sv:
+    input:
+        SFTP.remote(hostname + project_base + sv_combined)
+    output:
+        sv_combined
+    run:
+        shell("cp {input[0]} {output[0]}")
```

Comment:
New line is missing here, but would it make sense to also add the expression data syncing here?